### ISSUE TITLE
Show Gitcoin "loading" gif when token is still being sync'ed to the blockchain

### ISSUE THREAD

**Issue by octavioamu** [gitcoinco/web#2642]
_From @jasonrhaas on June 20, 2018 0:37_

Right now there is no feedback to users if we are waiting on the blockchain.

Should be able to re-use code from the Gitcoin App for this.

UPDATE:
Right now this will just be a confirmation page on any new Kudos clone, telling the user that the Kudos will be available on the Blockchain and Gitcoin Profile after a few minutes.

_Copied from original issue: gitcoinco/gitcoin-erc721#48_

**Comment by octavioamu** [comment]
_From @jasonrhaas on July 13, 2018 19:41_

Deferring this one until to the Gitcoin Application integration.

**Comment by octavioamu** [comment]
_From @jasonrhaas on September 20, 2018 20:11_

We might need to just add some notes to the confirmation page telling them that will be will have wait a few minutes for the transaction to mine.  After it is mined, it will show up in their profile.

**Comment by octavioamu** [comment]
_From @owocki on September 20, 2018 23:23_

if we copy the tips integration, this is already done https://github.com/gitcoinco/web/blob/master/app/dashboard/tip_views.py#L132-L133
